src/reconcile/rec_visibility.c

```diff
@@ -216,9 +216,10 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
     first_txn_upd = upd = last_upd = NULL;
     upd_memsize = 0;
     max_ts = WT_TS_NONE;
+    stable_ts = S2C(session)->txn_global.stable_timestamp;
```

Comment:
This needs to be the timestamp that was used by the last completed checkpoint, not the current stable timestamp.

Reply:
Why? I don't understand this.